spell/spell-crafter-mainnet-workflow.md

```diff
@@ -9,6 +9,11 @@ Repo: https://github.com/makerdao/spells-mainnet
 ![](https://ipfs.io/ipfs/QmUqCvy7c8Qmzn7yZ6D3353wTqCZ3VDAwQKYB37pJ2BjXb)
 
 ### Steps:
+* [ ] (Highly Desired, non-blocking) Make sure that the crafter relatied confirmations in the exec sheet are `Confirmed`
```

Comment:
I don't think it makes sense to add it to the beginning of the checklist, because first the content needs to be checked that it makes sense from the technical point of view. This is normally done by the end of the development stage, when the code is written, compiled, tests are written and executed. Before that I wouldn't give technical confirm.

Therefore, I propose to remove this disputable point from the PR (but you can submit it with the next PR)